src/components/Chip/Chip.stories.mdx

```diff
@@ -37,6 +37,31 @@ It can be used to display a small value attached to a component.
   </Story>
 </Canvas>
 
+### Appearance
+<Canvas>
+  <Story name="Positive-appearance">
```

Comment:
I think that one story with all 4 coloured chips in it is enough, we don't need separate example for each of them.

Reply:
Done :+1: